### ISSUE TITLE
Als beheerder wil ik met Docker kunnen werken

### ISSUE THREAD

**Issue by michielverhoef** [VNG-Realisatie/api-beheer#61]
...zodat ik weet hoe ik een referentie implementatie kan draaien en uitrollen.

**Definition of ready**

- [ ] Iedereen in het team begrijpt de user story
- [ ] Is klein genoeg voor een sprint
- [ ] Product Owner akkoord en voorzien van prio
- [ ] Voorzien van Acceptatiecriteria
- [ ] Voorzien van Definition of Done
- [ ] Voorzien van taken
- [ ] Vastgelegd Github en in geplaatst in kolom ready for sprint

**Definition of done**

**Algemeen**

Een werkwijze/middel/tools om ZDS-API goed in beheer te nemen.

**Specifiek**
 
- [ ] ...
- [ ] ...

**Oplossingsrichting**

...

**Acceptatiecriteria**

- [ ] Niet belemmerend werken voor beheer andere standaarden
- [ ] ...

**Taken**

- [ ] ... (@...)
- [ ] ... (@...)

**Comment by TCIMEddy** [comment]
@melsk-r ben je hier niet al mee bezig? Zo ja zet hem dan in de sprint op in Progress

**Comment by joeribekker** [comment]
Ik heb bij de demo API een kleine uitleg voor Docker toegevoegd.
Komt neer op bestandje aanmaken (docker-compose.yml) en 1 commando doen!

Zie: https://hub.docker.com/r/vngr/demo-api

**Comment by melsk-r** [comment]
> 
> 
> @melsk-r ben je hier niet al mee bezig? Zo ja zet hem dan in de sprint op in Progress

Klopt, in feite zijn we i.h.k.v. #57 hier al mee bezig. Ik zal hem op 'in Progress' zetten.

**Comment by TCIMEddy** [comment]
@wishalg Is het mogelijk dat Robert en Michiel een 2-daagse cursus of iets dergelijks kunnen volgen. Dan kunnen we deze user story concreter maken.
